src/ROUTES.ts

```diff
@@ -1552,6 +1550,12 @@ type Route = {
 
 type RoutesValidationError = 'Error: One or more routes defined within `ROUTES` have not correctly used `as const` in their `getRoute` function return value.';
 
+/**
+ * Represents all routes in the app as a union of literal strings.
```

Comment:
I added this comment, because I found the `RouteIsPlainString` quite confusing. After digging in `git blame` I found that this comment was originally there, but got removed in here:
https://github.com/Expensify/App/pull/40017/files#diff-8c80af31c75a0f4fcb67272bd2df5b42c9611c040ecc66584fcd6bde05dbbd78L749

I think having this comment is much better than not having it, because this type is not exported and not used anywhere, so it will 100% confuse others.